### PR TITLE
Add a FAQ on pushing from Jupyter

### PR DESCRIPTION
**Pull request by akarve**
<del>
# Description

# TODO

- [ ] Unit tests
- [ ] Automated tests (e.g. Preflight)
- [ ] Documentation
    - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [ ] JavaScript: basic explanation and screenshot of new features
- [ ] [Changelog](CHANGELOG.md) entry
</del>

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -62,3 +62,18 @@ a local machine or foreign region)—I/O is much faster.
...
+
+1. Push from outside your notebook with the CLI (`quilt3 push ....`). You can
+use Jupyter terminal for this.
+
+1. If you must push from within the notebook, save it first, and use the `%%capture`
+magic to prevent output from cells that change during push. This prevents a race
+condition between hashing the file and physically copying it to S3, during which
+time the file hash and size might change, due to progress bars and other text
+that `quilt3` writes to your notebook. Failing this precaution, `quilt3 install`
+may fail since the file in S3 contains may have a different hash than your local
```
Comment (sir-sigurd) [inline docs/FAQ.md:78]: I don't think `quilt3 install` verifies hashes of data.

**Review by sir-sigurd — COMMENTED** [review]
The problem doesn't look specific to quilt3 progress bars inside notebooks, so instead of adding such FAQ entry I think we should address somehow the underlying problem of "eventual consistency" of hashes and uploaded data.

File: docs/FAQ.md (hunk 1 above)
Comment (akarve) [reply docs/FAQ.md:78]: I think it check sizes then. The important thing: this race condition will cause `quilt3 install` to fail.

Comment (sir-sigurd) [reply docs/FAQ.md:78]: It doesn't look like so.

**Comment by drernie** [comment]
Doesn't seem to still be an issue. Closing.

**Pull request closed without merging**